analysisPlugin/src/main/scala/org/scalaclean/analysis/AnalysisModel.scala

```diff
@@ -168,12 +169,12 @@ sealed trait ModelSymbol extends HasModelCommon {
       println(s"duplicate of : $newCsvString")
     }
     this match {
-      case f : ModelField =>
-        f.constructorParam.foreach{ p =>
+      case f: ModelField =>
+        f.constructorParam.foreach { p =>
```

Comment:
From https://github.com/rorygraves/ScalaClean/pull/63#issuecomment-701928080

> @mkeskells Recommended approach is format on save (to avoid breaking IDE history:

It looks like this isn't being follow.  I see two ways forward: (a) break CI if it's not formatted or (b) setup sbt to format on compile and break CI if it's not formatted.  WDYT?  Any preferences?

Reply:
Yeah, I want to start enfocing this too - I'm really not sure which optin is best - I'm leaning towards (b)